tripal/src/TripalDBX/TripalDbx.php

```diff
@@ -2,10 +2,11 @@
 
 namespace Drupal\tripal\TripalDBX;
 
+use Drupal\Core\Database\Connection;
 use Drupal\tripal\TripalDBX\Exceptions\SchemaException;
 
 /**
- * Tripal DBX
+ * Tripal DBX.
```

Comment:
Lets use this opportunity to be more clear :-)


```suggestion
 * Service extending the Tripal DBX API to managing multiple schema.
```